CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 ### Changed
+* Make select error description more informative in case
+  of disabled/uninit storage.
```

Comment:
Nit: I prefer to mention issues and/or pull requests in a changelog and release notes to guide a curious reader, where to find more information. Usually a change *resolves* given issue, but not here. So I would write `(part of #229)` or `(part of #229, PR #297)` instead of just `(#229)` to make it clear.

Reply:
Okay. Done.